src/style/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+	margin: 0px;
+	border: 0px;
+	padding: 0px;
+}
+
+.main {
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+
+	background: rgb(250, 242, 215);
+}
+
+.nav {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-wrap: wrap;
+	background: rgba(251, 250, 248, 0.98);
+}
+
+.nav div {
+	position: relative;
+	margin: 3px 5px;
+	font-size: 14px;
+	color: rgb(0, 0, 0);
+}
+
+.nav div:nth-child(2n+1) {
+	font-size: 18px;
+}
+
+.nav div:nth-child(7n+5) {
+	font-size: 20px;
+} 
+
+.nav div:hover {
+	color: rgba(0, 0, 0, 0.5);
+	cursor: pointer;
+}
+
+.nav div .tooltiptext {
+  visibility: hidden;
+  background-color: rgba(0, 0, 0, 0.9);
+  color: rgb(255, 255, 255); 
+  text-align: center;
+  border-radius: 3px;
+  padding: 3px;
+
+  position: absolute;
+  z-index: 1;
+  top: 100%;
+  left: 50%;
+  margin-left: -60px;
+}
+
+.nav div:hover .tooltiptext {
+  visibility: visible;
+}
+
+.section {
+	padding: 20px 5%;
+	display: flex;
+	justify-content: space-around;
+	flex-wrap: wrap;
+}
+
+.section a {
+	width: 47%;
+	text-decoration: none;
+	color: #000000;
+}
+
+.section a .article {
+	height: 350px;
+	margin: 15px 0px;
+	display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+	background: linear-gradient(135deg, rgba(226,226,226,1) 0%, rgba(219,219,219,1) 30%, rgba(254,254,254,1) 100%);;
+  background-position: center;
+  background-size: cover;
+  box-shadow: -5px -5px 13px 0px rgba(0, 0, 0, 0.75);
```

Comment:
Please, follow same style for indents for all file. You can use some extension in your IDE for indents formatting